src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.ts

```diff
@@ -0,0 +1,258 @@
+      this.clusterDetailsArray = Object.values(clusters['config'])
+        .flat()
+        .filter((cluster) => cluster['cluster_alias'] !== 'local-cluster');
+      this.isMultiClusterConfigured = this.clusterDetailsArray.length > 0;
+      if (!this.isMultiClusterConfigured) {
+        this.stepTitles = ['Create Realm & Zonegroup', 'Create Zone', 'Export Multi-site token'];
+      } else {
+        this.selectedCluster = this.clusterDetailsArray[0]['name'];
+      }
+    });
+  }
+
+  createForm() {
+    this.multisiteSetupForm = new CdFormGroup({
+      realmName: new UntypedFormControl('default_realm', {
+        validators: [Validators.required]
+      }),
+      zonegroupName: new UntypedFormControl('default_zonegroup', {
+        validators: [Validators.required]
+      }),
+      zonegroup_endpoints: new UntypedFormControl(null, [Validators.required]),
+      zoneName: new UntypedFormControl('default_zone', {
+        validators: [Validators.required]
+      }),
+      zone_endpoints: new UntypedFormControl(null, {
+        validators: [Validators.required]
+      }),
+      username: new UntypedFormControl('default_system_user', {
+        validators: [Validators.required]
+      }),
+      cluster: new UntypedFormControl(null, {
+        validators: [Validators.required]
+      })
+    });
+
+    if (!this.isMultiClusterConfigured) {
+      this.exportTokenForm = new CdFormGroup({});
+    }
+  }
+
+  showSubmitButtonLabel() {
+    if (this.isMultiClusterConfigured) {
+      return !this.wizardStepsService.isLastStep()
+        ? this.actionLabels.NEXT
+        : $localize`Configure Multi-site`;
+    } else {
+      return !this.wizardStepsService.isLastStep() ? this.actionLabels.NEXT : $localize`Close`;
+    }
+  }
+
+  showCancelButtonLabel() {
+    return !this.wizardStepsService.isFirstStep()
+      ? this.actionLabels.BACK
+      : this.actionLabels.CANCEL;
+  }
+
+  onNextStep() {
+    if (!this.wizardStepsService.isLastStep()) {
+      this.wizardStepsService.getCurrentStep().subscribe((step: WizardStepModel) => {
+        this.currentStep = step;
+      });
+      if (this.currentStep.stepIndex === 2 && !this.isMultiClusterConfigured) {
+        this.loading = true;
+        this.onSubmit();
+      } else {
+        this.wizardStepsService.moveToNextStep();
+      }
+    } else {
+      this.loading = true;
+      this.onSubmit();
+    }
+  }
+
+  onSubmit() {
+    const values = this.multisiteSetupForm.value;
+    const realmName = values['realmName'];
+    const zonegroupName = values['zonegroupName'];
+    const zonegroupEndpoints = this.rgwEndpoints.value.join(',');
+    const zoneName = values['zoneName'];
+    const zoneEndpoints = this.rgwEndpoints.value.join(',');
+    const username = values['username'];
+    if (!this.isMultiClusterConfigured) {
+      if (this.wizardStepsService.isLastStep()) {
+        this.activeModal.close();
+        this.handleMultiClusterNavigation();
```

Comment:
Using `handleMultiClusterNavigation` when multicluster is not configured doesn't seem right

Reply:
renamed